modules/ROOT/pages/op-installing-tkn-on-linux.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="installing-tkn-on-linux"]
+
+= Installing {op-title} on Linux
+
+== Binary installation
+
+. Get the binary:
++
+----
+# curl -L https://mirror.openshift.com/pub/openshift-v4/clients/tkn/latest/tkn-linux-amd64 -o /usr/local/bin/tkn
+
+----
+. Set up the permissions:
++
+----
+# chmod +x /usr/local/bin/odo
+----
+
+== Tarball installation
+
+=== Linux AMD64
+
+. Get the `tar.gz`:
++
+----
+curl -LO https://github.com/tektoncd/cli/releases/download/v0.8.0/tkn_0.8.0_Linux_x86_64.tar.gz
+----
+
+. Extract `tkn` to your PATH:
```

Comment:
Extract the `tkn` compressed file and place it in a directory that is on your PATH. ? Suggest changing in all instances in the assembly.

Reply:
I have made changes as per this https://docs.openshift.com/container-platform/4.3/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli_cli-developer-commands

1. Download the compressed file
2. Extract the compressed file
3. Place it in a directory that is on your PATH